baselines/flwr_baselines/publications/adaptive_federated_optimization/cifar/utils.py

```diff
@@ -467,7 +467,9 @@ def on_fit_config(server_round: int) -> Dict[str, Scalar]:
 
 def get_cifar_eval_fn(
     path_original_dataset: Path, num_classes: int = 10
-) -> Callable[[Weights], Optional[Tuple[float, Dict[str, float]]]]:
+) -> Callable[
+    [int, NDArrays, Dict[str, Scalar]], Optional[Tuple[float, Dict[str, float]]]
```

Comment:
Should the type signature perhaps look like this?
```suggestion
    [int, NDArrays, Dict[str, Scalar]], Optional[Tuple[float, Dict[str, Scalar]]]
```